spec.bs

```diff
@@ -647,6 +647,31 @@ Each time a <{fencedframe}> navigates to a [=fenced frame config=], it is instan
 [=fenced frame config instance=], which governs the particular [=browsing context group=] inside the
 [=fenced navigable container/fenced navigable=].
 
+<h4 id=fenced-frame-config-use-cases>Use cases</h4>
+
+Rendering an ad created through an ad auction:
+
+An ad auction API runs an auction and determines a winning ad. Details about the winning ad must be
+hidden from the embedder, and the embedding context cannot be allowed to influence the environment
+of the <{fencedframe}>. Either of those will allow for information to flow across the fenced frame
+boundary, which can allow for colluding parties to join cross-site data and build a profile on the
+user. To prevent that, the API constructs a [=fenced frame config=] whose underlying [=URL=] is
+opaque to the embedding context. The [=fenced frame config=] is also constructed with restrictions
+on what the [=fenced frame config/container size=] and [=fenced frame config/content size=] of the
+frame must be and what the [=fenced frame config/effective enabled permissions|permissions policy=]
+of the frame must be, as those can be used as fingerprinting vectors.
+
+Displaying a personalized payment button:
+
+An e-commerce site embeds a <{fencedframe}> that has a "Pay now" button. This button includes the
+last 4 digits of the user's credit card number as is saved with the e-commerce platform. At first,
+the {{Document}} hosted in the <{fencedframe}> has no access to the data from the e-commerce
```

Comment:
"has no access to data from the e-commerce platform" is kinda vague. This is technically not true, since the document contents is very much "data from the e-commerce platform". I think you mean to say that it initially has no 1p cookie/storage (anything else?) access. Can you make this a little more concrete accordingly?